R/pr.R

```diff
@@ -237,7 +237,7 @@ pr_resume <- function(branch = NULL) {
     ui_bullets(c(
       "i" = "No branch specified ... looking up local branches and associated PRs."
     ))
-    default_branch <- git_default_branch()
+    default_branch <- guess_local_default_branch()
```

Comment:
I think it's ok to use a cheap check here because we're only using it to exclude branches, and this (along with the tweak below) allows `pr_resume()` to work offline.